client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -64,7 +64,7 @@ class SpeedUpSiteSettings extends Component {
 							moduleSlug="photon"
 							label={ translate( 'Serve images from our servers' ) }
 							description={ translate(
-								'Jetpack will optimize your images and server them from the server ' +
+								'Jetpack will optimize your images and serve them from the server ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Jetpack will optimize your images and server them from the server location nearest to your visitors. Using our global content delivery network will boost the loading speed of your site.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).